test/run-spec-tests.js

```diff
@@ -0,0 +1,49 @@
+import { Marked } from '../lib/marked.esm.js';
```

Comment:
So an extension package will just need to make their own version of this file to run spec tests, right?

I'm most interested in spec tests, but is there a similar capability for running unit tests?

Reply:
Ya, there is a [`getAllMarkedSpecTests()`](https://github.com/markedjs/testutils#getallmarkedspectests) method to get the marked specs.

Reply:
I'm not using that method in marked because we want to be able to run tests that are created in PRs.

Reply:
Here is an example PR for an extension to run all marked tests

 https://github.com/UziTech/marked-katex-extension/pull/255/files